flow360/log.py

```diff
@@ -289,7 +289,7 @@ def set_logging_file(
     filemode: str = "a",
     level: LogValue = DEFAULT_LEVEL,
     back_up_count: int = 10,
-    max_bytes: int = 10000,
+    max_bytes: int = 100000000,
```

Comment:
why changing this again? Is it a conflict?

Reply:
Accidentally changed when running tests to speed up the process before merging. Reverting in next commit to prevent conflict.

Reply:
Reverted in 644f8ee